tests/integrations/mcs/tso/server_test.go

```diff
@@ -373,3 +395,37 @@ func TestMetrics(t *testing.T) {
 	re.NoError(err)
 	re.Contains(string(output), "tso_server_info")
 }
+
+func (suite *CommonTestSuite) TestBootstrapDefaultKeyspaceGroup() {
+	re := suite.Require()
+
+	// check the default keyspace group
+	check := func() {
+		resp, err := http.Get(suite.pdLeader.GetServer().GetConfig().AdvertiseClientUrls + "/pd/api/v2/tso/keyspace-groups")
```

Comment:
This RESTful api returns keyspace groups by loading from etcd storage whenever it is invoked? If so, it might impact API server's performance when it's being processed.

Reply:
out of range of this pr. if could, we just have short discussion about it in this thread, but please ignore it for this pr.

Reply:
For now, it is. An optional choice is to return the group cached in the group manager.